src/module-vsbridge-indexer-catalog/Index/Mapping/AbstractMapping.php

```diff
@@ -13,16 +13,17 @@ abstract class AbstractMapping
     /**
      * @var array
      */
-    private $staticFieldMapping = [
-        'status' => FieldInterface::TYPE_INTEGER,
-        'visibility' => FieldInterface::TYPE_INTEGER,
-        'position' => FieldInterface::TYPE_LONG,
-        'level' => FieldInterface::TYPE_INTEGER,
-        'category_ids' => FieldInterface::TYPE_LONG,
-        'sku' => FieldInterface::TYPE_KEYWORD,
-        'url_path' => FieldInterface::TYPE_KEYWORD,
-        'url_key' => FieldInterface::TYPE_KEYWORD,
-    ];
+    protected $staticFieldMapping;
```

Comment:
Can we stay with `private` as it was ? Now we can use di.xml to add new fields to this.. so I don't think so we need to change it to `protected`.

I would add new method to access this variable, if needed, `getStaticFieldMapping` or `getStaticFields`

Reply:
It us up to you but:
1. Writing accessor methods only because we can is an overengineering. In private/protected context there is no real benefit for that.
2. If we keep this attribute private then it will be not override in children classes and therefore must be defined there too, in overriding classes, current and future. Something that can be missed.

I doubt that we should do so much to achieve the same result.
What do you think?

Reply:
I would leave it as private for now.

If someone need access to this variable in overrriding class or children classess I will think about a different approach.

Reply:
I made a change.